stylesheets/blue/components/_button_primary.scss

```diff
@@ -4,6 +4,7 @@
 //
 // default - Normal state
 // .ButtonPrimary--fullWidth - A button that takes the whole width of its parent
+// .ButtonPrimary--brightTurquoise - A button that uses the brightTurquoise color
```

Comment:
If there's a valid reasoning behind the name I'm all for it 👍 
Doesn't it have any restrictions regarding its hierarchical function - always primary? always secondary? can be both?